### PR TITLE
create AWS account for Tess

### PR DESCRIPTION
**Pull request by zaro0508**

### CODE REVIEW

**Review by ahayden — CHANGES_REQUESTED** [review]
negative

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ Organization:
         - !Ref ScipoolProdAccount
         - !Ref ScicompAccount
         - !Ref SandboxAccount
+        - !Ref TThyerAccount
```
Comment (ahayden) [inline org-formation/organization.yaml:30]: Let's rename this `MobileHealth-DataEngineeringAccount`

File: org-formation/organization.yaml [hunk 2]
```diff
@@ -258,6 +259,16 @@ Organization:
         Department: Platform
         Project: Infrastructure
 
+  TThyerAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-tthyer
```
Comment (ahayden) [inline org-formation/organization.yaml:265]: This should be `org-sagebase-mobilehealth-dataengineering`

File: org-formation/organization.yaml [hunk 3]
```diff
@@ -258,6 +259,16 @@ Organization:
...
+      RootEmail: tthyer@sagebase.org
+      Alias: org-sagebase-tthyer
```
Comment (ahayden) [inline org-formation/organization.yaml:267]: This should be `org-sagebase-mobilehealth-dataengineering`

**Review by ahayden — CHANGES_REQUESTED** [review]
This would be a dev account owned by Tess, but should be named after the project its for.

File: org-formation/organization.yaml [hunk 4]
```diff
@@ -248,3 +249,13 @@ Organization:
       Tags:
         Department: Platform
         Project: Infrastructure
+
+  MobileHealthDataEngineeringAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: mobilehealth-dataengineering
+      RootEmail: aws-mobilehealth-dataengineering@sagebase.org
```
Comment (ahayden) [inline org-formation/organization.yaml:257]: Nice- I forgot about this

**Review by ahayden — APPROVED** [review]
Thank you!

**Review by tthyer — APPROVED** [review]

**Pull request merged**